bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
It is been fixed in https://github.com/NetSys/bess/pull/500/commits/5e4a82d4967565185f0cd1c9860328da1e1b882c